### DISCUSSION TITLE
Unable to disabling Parallel function calling in Automatic function Invocation calling in Semantic kernel

### DISCUSSION THREAD

**Discussion by rakshahulle** [microsoft/semantic-kernel#9871] [Q&A]
I am trying to run function sin sequence but its not work in Automatic function invocation due to by default function calling is parallel. I have tried setting parallel_tool_call to false in python code but its still not working. Please guide me.

**Comment by sophialagerkranspandey** [comment]
Hi @rakshahulle, we wanted to provide you with documentation here: https://learn.microsoft.com/en-us/semantic-kernel/concepts/ai-services/chat-completion/function-calling/?pivots=programming-language-python#parallel-function-calls. This will depend on which model you are using as well.

**Comment by rakshahulle** [comment]
Which model is required?
